eppoclient/httpclient.go

```diff
@@ -23,22 +26,26 @@ type SDKParams struct {
 	sdkVersion string
 }
 
-func newHttpClient(baseUrl string, client *http.Client, sdkParams SDKParams) *httpClient {
+type HttpResponse struct {
+	Body string
```

Comment:
This should be `[]byte` to avoid copying the bytes when converting to/from string.
```suggestion
	Body []byte
```